packages/realm/src/Logger.ts

```diff
@@ -40,7 +167,60 @@ export enum NumericLogLevel {
  */
 export type Logger = (level: NumericLogLevel, message: string) => void;
 
-export type LoggerCallback = (level: LogLevel, message: string) => void;
+/**
+ * A callback passed to `Realm.setLogger`.
```

Comment:
Rather than tying this in the description to where it is used, I'd suggest only describing what it is (as `Realm.setLogger()` will already be referencing this).
```suggestion
 * A callback to be used as the logger.
```